visit-scheduler-api-specification.yaml

```diff
@@ -13,14 +13,16 @@ servers:
           - preprod   #Pre-production
           - Prod      #Production
 paths:
-  /visit-sessions/prison/{PrisonId}:
+  /bookable-visit-sessions-by-prison/{prisonId}:
     get:
```

Comment:
we can discuss these on wednesday but combining resource name and filter doesn't feel right.  
I thought we'd agreed to go with /visit-sessions?prison=prisonId

Reply:
Yep, I agree with Lucy on this, you could end up with some really long names if we follow this pattern.